selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestSshServiceClient.java

```diff
@@ -14,48 +14,95 @@
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
+import com.google.inject.name.Named;
 import java.util.List;
+import org.eclipse.che.api.core.ConflictException;
+import org.eclipse.che.api.core.NotFoundException;
 import org.eclipse.che.api.core.rest.HttpJsonRequestFactory;
 import org.eclipse.che.api.core.rest.HttpJsonResponse;
 import org.eclipse.che.api.ssh.shared.dto.GenerateSshPairRequest;
 import org.eclipse.che.api.ssh.shared.dto.SshPairDto;
 import org.eclipse.che.selenium.core.provider.TestApiEndpointUrlProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** @author Musienko Maxim */
 @Singleton
 public class TestSshServiceClient {
+  private static final Logger LOG = LoggerFactory.getLogger(TestSshServiceClient.class);
```

Comment:
use similar way to get logger in different places. I prefer this one.

Reply:
Could you, please, explain, which places did you mean?